### PR TITLE
preserve int division

### PR DESCRIPTION
**Pull request by nickpell**
@dimagi/py3

### CODE REVIEW

File: corehq/apps/callcenter/checks.py [hunk 1]
```diff
@@ -60,7 +61,7 @@ def _compile_stats(name, ucr_count, es_count):
     total = ucr_count + es_count
 
     def _percent(count):
-        return 0 if total == 0 else int(100 * count / total)
+        return 0 if total == 0 else int(100 * count // total)
```
Comment (emord) [inline corehq/apps/callcenter/checks.py]: Do these still need to be wrapped with `int()`?

**Review by millerdev — APPROVED** [review]
@emord's question is valid

File: corehq/apps/callcenter/checks.py (hunk 1 above)
Comment (nickpell) [reply corehq/apps/callcenter/checks.py]: Probably not - my reasoning was that if ```count``` was ever a floating point value then ``` count // total``` would be an integer with floating point type (i.e. ```3.0```) but on closer inspection it looks like ```count``` is always an ```int```.  I'll simplify the code in a followup PR.

Comment (nickpell) [reply corehq/apps/callcenter/checks.py]: https://github.com/dimagi/commcare-hq/pull/19441

**Pull request merged**